cargo-dist/src/init.rs

```diff
@@ -916,6 +918,13 @@ fn apply_dist_to_metadata(metadata: &mut toml_edit::Item, meta: &DistMetadata) {
         unix_archive.map(|a| a.ext()),
     );
 
+    apply_optional_value(
+        table,
+        "npm-package",
+        "# The npm package should have this name\n",
```

Comment:
Let's make the wording consistent with Homebrew

```suggestion
        "# Customize the npm package name\n",
```